hnn_core/batch_simulate.py

```diff
@@ -70,10 +73,22 @@ def __init__(self, set_params, net_name='jones', tstop=170,
         summary_func : callable, optional
             A function to calculate summary statistics from the simulation
             results. Default is None.
+        save_dpl : bool
+            If True, save dipole results. Default: True.
+        save_spiking : bool
+            If True, save spiking results. Default: False.
+        save_LFP : bool
+            If True, save local field potential (LFP) results. Default: False.
+        save_voltages : bool
+            If True, save voltages results. Default: False.
+        save_currents : bool
+            If True, save currents results. Default: False.
+        save_calcium : bool
+            If True, save calcium concentrations. Default: False.
         Notes
         -----
         When `save_output=True`, the saved files will appear as
-        `sim_run_{start_idx}-{end_idx}.npy` in the specified `save_folder`
+        `sim_run_{start_idx}-{end_idx}.npz` in the specified `save_folder`
```

Comment:
I remember finding a workaround at some point for combining`spike_<times/spikes/gids>` in a single `.npy` file, however this seems a bit more explicit